client/src/App.js

```diff
@@ -2,15 +2,13 @@ import React from "react";
 import { BrowserRouter, Route } from "react-router-dom";
 import Navigation from "./components/Navigation/Navigation";
 import Login from "./routes/LoginRoute/Login";
-import Gallery from "./routes/GalleryRoute/Gallery";
 import Workspace from "./routes/WorkspaceRoute/Workspace";
 
 const App = () => {
   return (
     <BrowserRouter>
       <Navigation />
       <Route path="/" exact={true} component={Login} />
-      {/* <Route path="/first" component={Gallery} /> */}
```

Comment:
Do we need curly braces here?

Reply:
When I comment something out, Prettier automatically adds curly braces. I can remove them though!

Reply:
Done 👍🏽